components/DetailPageHeader.js

```diff
@@ -111,24 +113,32 @@ class DetailPageHeader extends Component {
     let tabs;
     let overflow;
     let icon;
+    let tag;
+    let link;
 
     Children.map(children, child => {
-      if (child.type === Breadcrumb) {
-        breadcrumb = child;
+      if (child != null && child.type != null) {
+        switch (child.type) {
```

Comment:
Changed the `if` statements to a `switch` since I thought that would be more appropriate for the logic.